lib/jsdom/browser/Window.js

```diff
@@ -556,6 +556,97 @@ function Window(options) {
 
     forEachSheetRuleOfElement(elt, setPropertiesFromRule);
 
+    /**
+     * properties for which `getResolvedValue` is implemented which is less than
+     * every supported property according to step 5
+     * @type Record<string, { inherited: boolean; initial: unknown; computedValue: 'as-specified' }>
+     */
+    const properties = {
+      visibility: {
+        inherited: true,
+        initial: "visible",
+        computedValue: "as-specified"
+      }
+    };
+
+    /**
+     * Naive implementation of https://www.w3.org/TR/CSS2/cascade.html#cascade
```

Comment:
Can you change all the spec references to https://drafts.csswg.org/css2/ ? And make sure that the algorithms haven't changed in important ways :).

Reply:
This should actually now be https://drafts.csswg.org/css-cascade/#cascading.

> This module replaces and extends the rules for assigning property values, cascading, and inheritance defined in [[CSS2]](https://www.w3.org/TR/CSS2/) chapter 6.
>
> – https://drafts.csswg.org/css-cascade/#placement

Reply:
Couldn't detect changes. Only parts that are not implemented yet.